Services/Common/OJS.Services.Common.Models/Contests/Results/ContestResultsModel.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace OJS.Services.Common.Models.Contests.Results;
+
+using AutoMapper;
+using OJS.Data.Models.Contests;
+using SoftUni.AutoMapper.Infrastructure.Models;
+using System.Linq;
+
+public class ContestResultsModel : IMapExplicitly
+{
+    public Contest Contest { get; set; } = new Contest();
+
+    public bool Official { get; set; }
+
+    public bool IsUserAdminOrLecturer { get; set; }
+
+    public bool IsFullResults { get; set; }
+
+    public int? TotalResultsCount { get; set; }
+
+    public bool IsExportResults { get; set; }
+
+    public int Page { get; set; } = 1;
+
+    public int ItemsInPage { get; set; } = int.MaxValue;
+
+    public void RegisterMappings(IProfileExpression configuration) =>
+        configuration.CreateMap<ContestResultsViewModel, ContestResultsModel>()
+            .ForPath(
+                d => d.Contest.Name,
+                opt => opt.MapFrom(s => s.Name))
+            .ForPath(
+            d => d.Official,
+            opt => opt.MapFrom(s => s.IsCompete))
+            .ForPath(
+            d => d.Contest.Type,
```

Comment:
Fix formatting

Reply:
After discussing with you, we came to the conclusion that it’s good.